languages/elixir/concepts/lists/about.md

```diff
@@ -26,6 +35,8 @@ list = [2, 1]
 # => true
 ```
 
+Appending an element to a list during iteration is considered an anti-pattern. We can achieve the same result by prepending an element to the reversed list, and then reversing the result.
```

Comment:
Maybe expand on why it is an anti pattern, I think it's good to point out a potential solution as you have.